Samples/7.ServerList/Program.cs

```diff
@@ -3,6 +3,7 @@
 using System.Net;
 using SteamKit2;
 using SteamKit2.Internal;
+using SteamKit2.Discovery;
 
```

Comment:
Nit: Sort usings.
Also, in some places in this PR `System.*` is first, in other places it's purely alphabetical. Pick a style and stick to it.